certbot/certbot/crypto_util.py

```diff
@@ -174,18 +184,39 @@ def import_csr_file(csrfile, data):
     return PEM, util.CSR(file=csrfile, data=data_pem, form="pem"), domains
 
 
-def make_key(bits):
-    """Generate PEM encoded RSA key.
+def make_key(bits=1024, key_type="rsa", elliptic_curve=None):
+    """Generate PEM encoded RSA|EC key.
 
-    :param int bits: Number of bits, at least 1024.
+    :param int bits: Number of bits, at least 1024 for RSA.
```

Comment:
As in `init_save_key`, we should add for `bits` and `ec_curve` that they apply respectively for key type `rsa` and `ecdsa`.